smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -460,8 +460,8 @@ void updateDevices() {
 			childDevice.sendEvent(name: "saturation", value: device.color.saturation * 100)
 		}
 		childDevice.sendEvent(name: "label", value: device.label)
-		childDevice.sendEvent(name: "level", value: Math.round((device.brightness ?: 1) * 100))
-		childDevice.sendEvent(name: "switch.setLevel", value: Math.round((device.brightness ?: 1) * 100))
+		childDevice.sendEvent(name: "level", value: Math.round((device.brightness != null ? device.brightness : 1) * 100))
+		// childDevice.sendEvent(name: "switch.setLevel", value: Math.round((device.brightness ?: 1) * 100))//----------------------------------------------------------
```

Comment:
Whats up with //----------------------------------------------------------

Reply:
Line 464 needs to be deleted. I explained setLevel being a command vs. level being an attribute